src/client/collectors.py

```diff
@@ -237,32 +237,34 @@ class Parameters(CollectorConfig.Parameters):
 
 @inherit_docstrings
 class Captcha(CollectorConfig, spec_value=CollectorConfig.Type.CAPTCHA):
-    """Captchas provide a high level of protection from robots
+    """Collects captcha statistics for every Toloker.
 
-    Used with conditions:
-    * StoredResultsCount - How many times the Toloker entered captcha.
-    * SuccessRate - Percentage of correct answers of the Toloker to the captcha.
-    * FailRate - Percentage of wrong answers of the Toloker to the captcha.
+    Captcha provides an advanced protection against robots. It is used with conditions:
+    * [StoredResultsCount](toloka.client.conditions.StoredResultsCount.md) — How many times the Toloker entered a captcha.
+    * [SuccessRate](toloka.client.conditions.SuccessRate.md) — The percentage of solved captchas.
+    * [FailRate](toloka.client.conditions.FailRate.md) — The percentage of unsolved captchas.
 
-    Used with actions:
-    * RestrictionV2 - Block access to projects or pools.
-    * ApproveAllAssignments - Approve all replies from the Toloker.
-    * RejectAllAssignments - Reject all replies from the Toloker.
-    * SetSkill - Set Toloker's skill value.
-    * SetSkillFromOutputField - Set Toloker's skill value from source.
+    The collector can be used with actions:
+    * [RestrictionV2](toloka.client.actions.RestrictionV2.md) blocks access to projects or pools.
+    * [ApproveAllAssignments](toloka.client.actions.ApproveAllAssignments.md) accepts all Toloker's assignments.
+    * [RejectAllAssignments](toloka.client.actions.RejectAllAssignments.md) rejects all Toloker's assignments.
+    * [SetSkill](toloka.client.actions.SetSkill.md) sets Toloker's skill value.
+    * [SetSkillFromOutputField](toloka.client.actions.SetSkillFromOutputField.md) sets Toloker's skill value using an output field.
 
     Attributes:
-        parameters.history_size: The number of times the Toloker was shown a captcha recently.
+        parameters.history_size: The maximum number of recent captchas used to calculate the statistics.
+            If `history_size` is omitted, all captchas are counted.
 
     Example:
-        How to ban a Toloker in this project if he mistakes in captcha.
+        The example shows how to block Toloker's access to the project for 15 days if they solve 60% of captchas or less.
```

Comment:
Should we add "if a Toloker completed captcha at least 3 times"?

Reply:
Added